shacl12-profiling/index.html

```diff
@@ -898,26 +921,34 @@ <h3>Profile Part Roles</h3>
     </p>
     <p>
       Referencing [[[dx-prof]]]'s <a href="https://www.w3.org/TR/dx-prof/#resource-roles-vocab">vocabulary of <em>Resource
-      Role Instances</em></a>, it seems clear that SHACL graphs can be applied to the role of <a
-            href="https://www.w3.org/TR/dx-prof/#Role:validation">Validation</a> within a profile and perhaps also to
-      the role of <a href="https://www.w3.org/TR/dx-prof/#Role:specification">Specification</a>, since SHACL can be used
+      Role Instances</em></a>, it seems clear that
+      SHACL graphs can perform the role of <a
+            href="https://www.w3.org/TR/dx-prof/#Role:validation">Validation</a> within a profile and perhaps also the role of
+      <a href="https://www.w3.org/TR/dx-prof/#Role:specification">Specification</a>, since SHACL can be used
       to declare Node and Property Shapes just as OWL can be used to declare Classes and Properties.
     </p>
     <p>
-      SHACL resources can also document the constraints they implement validation shapes for, therefore they can also be
-      applied to the <a href="https://www.w3.org/TR/dx-prof/#Role:constraints">Constraints</a> role.
+      SHACL shapes naturally constrain models and data, therefore if used to indicate how a profile contrains a thing
+      profiles - a standard or another profile - they could perform the role of
+      <a href="https://www.w3.org/TR/dx-prof/#Role:constraints">Constraints</a>.
     </p>
     <p>
-      They could also be applied to the <a href="https://www.w3.org/TR/dx-prof/#Role:mapping">Mapping</a> role, if SHACL
+      They could also perform the role of <a href="https://www.w3.org/TR/dx-prof/#Role:mapping">Mapping</a>, if SHACL
       Rules are implemented to transform data from one model to another.
     </p>
-    <p>
-      SHACL lists of values required for use with a model could be applied to the <a
-            href="https://www.w3.org/TR/dx-prof/#Role:vocabulary">Vocabulary</a> role.
-    </p>
   </section>
   <section id="hierarchies">
     <h3>Profile Hierarchies</h3>
+    <p>
+      Profile hierarchies are sets of profiles where some profile the others. As per the rules of the [[[dx-prof]]], if
```

Comment:
```suggestion
      Profile hierarchies are arrangements in which a profile is a specialization of another, forming a chain where more specific profiles inherit constraints from broader ones. As per the rules of the [[[dx-prof]]], if
```

Reply:
Minor tweaks to @scotthenninger's suggestion
```suggestion
      Profile hierarchies are arrangements in which one profile is a specialization
      of another, forming a chain where more specific profiles inherit constraints
      from broader ones. As per the rules of the [[[dx-prof]]], if
```